tests/lightning_tensor/test_gates_and_expval.py

```diff
@@ -138,6 +138,7 @@ def circuit_ansatz(params, wires):
     qml.ECR(wires=[wires[1], wires[3]])
 
 
+@pytest.mark.parametrize("method", [{"method": "mps", "max_bond_dim": 128}, {"method": "tn"}])
```

Comment:
Could we apply this config to the following test unit as well?

Reply:
Done!